pkg/webui/components/button/button.styl

```diff
@@ -13,69 +13,67 @@
 // limitations under the License.
 
 .button
+  // Without secondary or primary style,
+  // the button should not apply any styling.
+  // This is to cater for special situations
+  // where the button is used as a container.
```

Comment:
See my other comment, this way of styling also avoids a lot of repetition and style overlaps in the button component.